src/scripts.js

```diff
@@ -3,31 +3,58 @@
 let greeting = document.querySelector('.greeting');
 let stepGoalComparison = document.querySelector('.you-vs-others-data');
 
+
+let userCardID = document.querySelector('.user-card-id');
+let userCardName= document.querySelector('.user-card-name');
+let userCardAddress = document.querySelector('.user-card-address');
+let userCardEmail = document.querySelector('.user-card-email');
+let userCardStrideLength = document.querySelector('.user-card-stride-length');
+let userCardDailyStepGoal = document.querySelector('.user-card-daily-step-goal');
+let userCardFriends = document.querySelector('.user-card-friends');
+
+
 let userRepository = new UserRepository(userData);
+let user = createUser();
 
 ///// event listeners /////
 
 window.addEventListener('load', compareStepGoals);
+window.addEventListener('load', createUser);
+window.addEventListener('load', welcomeUser);
+window.addEventListener('load', showUserCardData);
```

Comment:
I think setting these event listeners on the `window` is a great idea because all of these functions need to run on load or refresh, maybe we can make this look cleaner by adding a named function to the window, which invokes all of the necessary functions written on lines 21-24.  I can show you what I mean by this the next time we work together!

Reply:
I did this on my Mod1 final and it looked much cleaner! I think we should definitely look into this.